### PR TITLE
Function works, tests passing

### PR DESCRIPTION
**Pull request by SianCulligan**

### CODE REVIEW

File: code_challenges/401/llMerge/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Write a function called mergeLists which takes two linked lists as arguments. Zip the two linked lists together into one so that the nodes alternate between the two lists and return a reference to the head of the zipped list. Try and keep additional space down to O(1). You have access to the Node class and all the properties on the Linked List class as well as the methods created in previous challenges.
+
+### Approach & Efficiency
+I approached the problem by first thinking through the basic problem (zipping 2 lists) & building the function to handle edge cases by checking the length of the given linked list.
+
+## Question:
+- I couldn't figure out how to feed 2 linked lists through as arguments, so for the tests, I gave the function 2 arrays that the function could make into a linked list. I'd love to know the right way to do this - can someone please help point me to a resource or help me out?
+
+### Solution
+[White-board](./assets/llmerge.jpeg)</br></br>
```
Comment (lizkavalski) [inline code_challenges/401/llMerge/README.md:16]: capitalize the  jpeg
